### PR TITLE
Trivial bugfix: don't prohibit username with spaces

### PR DESCRIPTION
**Pull request by ujihisa**
* Problem: You can't input username with space like "7 days to die" or "Vim script"
    * It transforms into "7daystodie" or "Vimscript", which do not make sense
* Solution: Don't remove spaces yours input

### CODE REVIEW

**Comment by pankona** [comment]
@ujihisa 

good morning!

#98 intoduces changes the way to handle white spaces. FYI.

So I think `a b` and `   a b   ` should treat as same name here, just only removing `trim` from `setUsername` is not suitable in this case.

Here's my idea to improve this patch. How about this?
- Remove trim as this patch. It's okay.
- Additionally, do `trim` on submit

**Comment by ujihisa** [comment]
Makes sense. Let me try that!

File: src/pages/AppContainer.tsx [hunk 1]
```diff
@@ -107,7 +107,7 @@ const AppContainer: React.FunctionComponent = () => {
 
   const onUsernameChange = React.useCallback(
     (event: React.ChangeEvent<HTMLInputElement>) => {
-      setUsername(event.currentTarget.value.trim());
```
Comment (kachick) [inline src/pages/AppContainer.tsx:110]: >Problem: You can't input username with space like "7 days to die" or "Vim script"
It transforms into "7daystodie" or "Vimscript", which do not make sense

It sounds weird to me 🤔 `trim` just trims terminations.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim

スクリーンショット 2021-05-09 11 08 49
スクリーンショット 2021-05-09 11 08 55

Comment (pankona) [reply src/pages/AppContainer.tsx:110]: @kachick 

currently `trim` is called for every input change, it means we can't input `7 `, it is the way to `7 days to die`. I think it is not convenience.

Comment (kachick) [reply src/pages/AppContainer.tsx:110]: Ah understood! I always copy and paste... So I didn't notice the issue 🙇

**Comment by ujihisa** [comment]
Fixed! See the latest commit. I tested locally and seemed fine :+1:

**Comment by pankona** [comment]
@ujihisa 

seems okay but `a b` and `a b  ` can register now.
Could you please pretend from registering it as duplicated? I think https://github.com/mobu-of-the-world/mobu/blob/ef2250d6bfd2a13c35bbbb0f4536acdae520c490/src/pages/AppContainer.tsx#L39 may be updated as well.

**Comment by ujihisa** [comment]
Great point, that's right!

I fixed at c4c19a736f489ba99d75f50f0cf39251488619a4 now. I could just add `.trim()` against the `blankStringsPattern.test(username)`, but I thought this would be nicer.

**Review by pankona — APPROVED** [review]
Frawless!

**Comment by pankona** [comment]
Thanks! It's time to ship it! :shipit:

**Pull request merged**